shared/templates/static/ansible/require_smb_client_signing.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+- name: Check if /etc/samba/smb.conf exists
+  stat:
+    path: /etc/samba/smb.conf
+  register: st_smb
```

Comment:
Ah, you are right, since it's not `shell` and it already has both check and execution modes it doesn't need `check_mode no`